SteakBot.Core/Objects/MemeCommand.cs

```diff
@@ -19,5 +20,57 @@ public MemeCommand(MemeResultType resultType, string name, string value, string
             Value = value;
             Description = description;
         }
+
+        #region IEquitable
+
+        public override bool Equals(object other)
+        {
+            var item = other as MemeCommand;
+            return item != null && Equals(item);
+        }
+
+        public bool Equals(MemeCommand other)
+        {
+            if (ReferenceEquals(other, null))
+            {
+                return false;
+            }
+
+            return (object.ReferenceEquals(this.ResultType, other.ResultType)
```

Comment:
object.ReferenceEquals is always false because it is called on value type. C'MON @naps250